includes/admin/onboarding/steps/step-products.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Onboarding Wizard Products Step.
+ *
+ * @package     EDD
+ * @subpackage  Onboarding
+ * @copyright   Copyright (c) 2022, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.2
+ */
+
+namespace EDD\Onboarding\Steps\Products;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+use EDD\Onboarding\Helpers;
+
+/**
+ * Initialize step.
+ *
+ * @since 3.2
+ */
+function initialize() {
+	add_action( 'wp_ajax_edd_onboarding_create_product', __NAMESPACE__ . '\create_product' );
+}
+
+/**
+ * Ajax callback for creating a product.
+ *
+ * @since 3.2
+ */
+function create_product() {
+	if ( ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'edd_onboarding_wizard' ) ) {
+		exit();
+	}
+
+	if ( ! current_user_can( 'edit_products' ) ) {
+		return;
+	}
+
+	$response = array( 'success' => false );
+
+	// Prepare product post details.
+	$product = array(
+		'post_title'  => wp_strip_all_tags( $_REQUEST['product_title'] ),
+		'post_status' => 'draft',
+		'post_type'   => 'download',
+	);
+
+	// Insert the product into the database.
+	$post_id = wp_insert_post( $product );
+	if ( $post_id ) {
+		$post = get_post( $post_id );
+
+		// Save meta fields.
+		edd_download_meta_box_fields_save( $post_id, $post );
+
+		// Set featured image.
+		if ( ! empty( $_REQUEST['product_image_id'] ) ) {
+			set_post_thumbnail( $post_id, absint( $_REQUEST['product_image_id'] ) );
+		}
+
+		$response['success']      = true;
+		$response['redirect_url'] = get_edit_post_link( $post_id );
+	}
+
+	wp_send_json( $response );
+	exit;
+}
+
+/**
+ * Get step view.
+ *
+ * @since 3.2
+ */
+function step_html() {
+	$currency_position  = edd_get_option( 'currency_position', 'before' );
+	ob_start();
+	?>
+	<form method="post" class="edd-onboarding__create-product-form">
+		<input type="hidden" name="_edd_product_type" value="0">
+		<table class="form-table" role="presentation">
+			<tbody>
+				<tr>
+					<th scope="row"><h3><?php echo esc_html_e( 'Product details', 'easy-digital-downloads' ); ?></h3></th>
```

Comment:
I'm wondering if it would be possible to make the `<th>` span both columns and include the tooltip if we are committed to having that. I think we have them separate on the settings page because of the settings API, but here it is causing some hiccups with responsiveness (because of all the padding, the media queries feel very different) and since we have total control over the output, I don't think it's necessary to have both (for these headings, not for the rest).